public/app/features/dashboard/components/PanelEditor/FieldConfigEditor.tsx

```diff
@@ -190,7 +191,18 @@ export const DefaultFieldConfigEditor: React.FC<Props> = ({ data, onChange, conf
             onChange={v => setDefaultValue(item.path, v, item.isCustom)}
             context={{
               data,
-              getSuggestions: (scope?: VariableSuggestionsScope) => getDataLinksVariableSuggestions(data, scope),
+              getSuggestions: (scope?: VariableSuggestionsScope) => {
+                if (item.settings?.getSuggestions) {
+                  return item.settings.getSuggestions(
+                    plugin,
+                    data,
+                    getTemplateSrv().getVariables.bind(getTemplateSrv()),
+                    scope
+                  );
+                }
```

Comment:
Moved suggestions providers(i.e. getDataLinksVariableSuggestions) to grafana-data so that the configuration can be configured via declarative options API.

@ryantxu @torkelo do you think we should provide some default suggestions if no config is provided? Should those maybe be merged with the ones provided with the API?

Reply:
Yes, default behavior for templates! -- ideally just a flag to enable/disable since the standard variables should always be the same